lib/src/over_react_test/common_component_util.dart

```diff
@@ -406,25 +409,67 @@ void testRequiredProps(BuilderOnlyUiFactory factory, dynamic childrenFactory())
   });
 
   test('throws when the required prop is not set or is null', () {
```

Comment:
This message isn't accurate when `useComponent2Tests` is true, since it doesn't throw. Could we do two separate test blocks?

Reply:
Yep, good catch!